Harivigneshkumar/Contact Form 1/index.html

```diff
@@ -0,0 +1,276 @@
+        }
+        #save:hover{
+          background-color: rgb(85, 134, 12);
+        }
+        #deleteRow:hover{
+          background-color: rgb(211, 55, 55);
+        }
+        #edit:hover{
+          background-color:#3373b3;
+        }
+        button{
+          color:cadetblue;
+        }
+        
+    </style>
+</head>
+<body>
+   <center>
+    <div id="table">
+        <table id="table1" contenteditable="true">
+            <tr>
+                <th>First Name</th>
+                <th>Last Name</th>
+                <th>Email ID</th>
+                <th>Date of Birth</th>
+                <th>Website</th>
+                <th>Mobile Number</th>
+                <th>Work</th>
+                <th>Gender</th>
+                <th>Address</th>
+                <th>Group</th>
+                <th>Messager</th>
+                <th>State</th>
+            </tr>
+            <!--<tr>
+               <td><span id="data"></span></td>
+               <td><span id="data1"></span></td>
+               <td><span id="data2"></span></td>
+               <td><span id="data3"></span></td>
+               <td><span id="data4"></span></td>
+               <td><span id="data5"></span></td>
+               <td><span id="data6"></span></td>
+               <td><span id="data7"></span></td>
+               <td><span id="data8"></span></td>
+               <td><span id="data9"></span></td>
+               <td><span id="data10"></span></td>
+               <td><span id="data11"></span></td>
+            </tr>-->
+            
+ 
+    <!--<span id="data"></span>-->
+    <script>
+        
+
+       /* const myobjectString = localStorage.getItem('txtvalue');
+        const myobject = JSON.parse(myobjectString);
+        document.getElementById("data").innerHTML=*/
+        /*document.write('<tr><td>'+ obj.fname+'</td><td>'+obj.lname+'</td><td>'+obj.email+'</td><td>'+obj.dob+'</td><td>'+obj.website+'</td><td>'+
+            obj.mobilenumber+'</td><td>'+obj.Address+'</td><td>'+obj.work+'</td><td>'+obj.Groups+'</td><td>'+obj.messager+
+                '</td><td>'+obj.state+'</td><td>'+obj.Gender+'</td></tr>');*/
+         
+
+        /*document.getElementById("data").innerHTML=obj.fname;
+        document.getElementById("data1").innerHTML=obj.lname;
+        document.getElementById("data2").innerHTML=obj.email;
+        document.getElementById("data3").innerHTML=obj.dob;
+        document.getElementById("data4").innerHTML=obj.website;
+        document.getElementById("data5").innerHTML=obj.mobilenumber;
+        document.getElementById("data6").innerHTML=obj.work;
+        document.getElementById("data7").innerHTML=obj.Gender;
+        document.getElementById("data8").innerHTML=obj.Address;
+        document.getElementById("data9").innerHTML=obj.Groups;
+        document.getElementById("data10").innerHTML=obj.messager;
+        document.getElementById("data11").innerHTML=obj.state;*/
+
+        let text = localStorage.getItem("testJSON");
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.